addon/-private/state-machine.js

```diff
@@ -80,9 +80,19 @@ export default EmberObject.extend({
     return transitions[currentStep];
   },
 
-  next() {
```

Comment:
I'm a bit confused -- why did `next()` get removed here?

Reply:
Because I didn't find any usage of this method (apart from tests).

Reply:
Gotcha, sounds good